src/EdFi.AnalyticsMiddleTier.DataStandard32/Asmt/PostgreSQL/0014-View-StudentAssessmentFact-Create.sql

```diff
@@ -0,0 +1,113 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
```

Comment:
If you need an ods in postgres there is one here: https://techdocs.ed-fi.org/display/EDFITOOLS/Testing+With+Sample+Data
Make sure AMT can be installed on Postgres.